app.js

```diff
@@ -1,18 +1,23 @@
-const express = require('express');
+const express = require("express");
 const app = express();
+const endPoints = require("./endpoints.json")
 
-const{ getTopics } = require('./controllers/topics.controller')
+const { getTopics } = require("./controllers/topics.controller");
 
-app.get('/api/topics', getTopics);
+app.get("/api/topics", getTopics);
 
+app.get("/api", (req, res) => {
+    console.log(endPoints)
+    console.log({endPoints},"<<<<")
```

Comment:
Are these console logs serving any purpose? Make sure to remove these types of console logs before submitting pull requests.